src/Files.App/Strings/en-GB/Resources.resw

```diff
@@ -2935,4 +2940,10 @@
   <data name="SortBy" xml:space="preserve">
     <value>Sort by</value>
   </data>
+  <data name="PropertiesDialogIsRunAsAdmin.Text" xml:space="preserve">
+    <value>Run as administrator</value>
+  </data>
+  <data name="RunAsAdmin" xml:space="preserve">
+    <value>Run as administrator</value>
+  </data>
```

Comment:
Can you revert the changes to this file? Translations are setup to sync from Crowdin.

Reply:
Oh ok